jwst/master_background/master_background_step.py

```diff
@@ -0,0 +1,86 @@
+from ..stpipe import Step
+from .. import datamodels
+
+
+__all__ = ["MasterBackgroundStep"]
+
+
+class MasterBackgroundStep(Step):
+    """
+    MasterBackgroundStep:  Compute and subtract master background from spectra
+    """
+
+    spec = """
+        user_background = str(default=None) # Path to user-supplied master background
+        save_background = boolean(default=False) # Save computed master background
+    """
+
+
+    def process(self, input):
+
+        """
+        Compute a master background from N > 1 dedicated background exposures
```

Comment:
Why must there be more than one background exposure?

Reply:
Ah, so there could be just one dedicated background exposure in an association?  Hardly a "master" background then, but I suppose we should handle it, right?

Reply:
Correct.